diff --git a/node_modules/react-native-radio-buttons-group/lib/RadioButton.tsx b/node_modules/react-native-radio-buttons-group/lib/RadioButton.tsx
index 9e95130..84fb019 100644
--- a/node_modules/react-native-radio-buttons-group/lib/RadioButton.tsx
+++ b/node_modules/react-native-radio-buttons-group/lib/RadioButton.tsx
@@ -100,7 +100,8 @@ export default function RadioButton({
 const styles = StyleSheet.create({
   container: {
     alignItems: 'center',
-    marginHorizontal: 10,
+    marginHorizontal: 0,
+    padding: 5,
     marginVertical: 5,
   },
   border: {
